devel/udrone/files/etc/init.d/udrone

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh /etc/rc.common
+
+START=91
+USE_PROCD=1
+
+start_service() {
+	procd_open_instance "udrone"
+	procd_set_param command "/usr/sbin/udrone"
+	procd_append_param command "$(uci get network.lan.ifname)"
```

Comment:
I don't know that calling uci is ideal here. Maybe one of the network_ functions in network.sh.